src/nvim/eval/encode.c

```diff
@@ -807,7 +821,9 @@ char *encode_tv2string(typval_T *tv, size_t *len)
 {
   garray_T ga;
   ga_init(&ga, (int)sizeof(char), 80);
-  encode_vim_to_string(&ga, tv, "encode_tv2string() argument");
+  const int evs_ret = encode_vim_to_string(&ga, tv,
+                                           "encode_tv2string() argument");
+  (void)evs_ret;
```

Comment:
AFAIR `(void)func()` does not always work. Though I guess I can add assert here and to tv2echo (these may give error message sometimes, but not actually fail) and check and free the return in tv2json: last one in case of errors currently just returns incomplete output.